docs/containers.conf.5.md

```diff
@@ -467,7 +467,7 @@ and pods are visible.
 
 Path to the slirp4netns binary.
 
-**network_cmd_options**=[]
+**network_cmd_options**=["enable_ipv6=true",]
```

Comment:
Can you add the other slirp4netns options to this man page, https://github.com/containers/podman/blob/main/docs/source/markdown/podman-create.1.md#--networkmode---net